spec/dummy/docs/components/README.md

```diff
@@ -0,0 +1,8 @@
+# Core Components
+
+This file exists here currently just so that the tests for `Matestack::Core::Generators::ComponentGenerator` can run as they expect such a file to exist.
```

Comment:
@jonasjabari this is a ... very pragmatic solution but potentially not the best one. Please check the commit message :)